cmd/whitesourceExecuteScan.go

```diff
@@ -3,6 +3,9 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/SAP/jenkins-library/pkg/maven"
+	"github.com/SAP/jenkins-library/pkg/npm"
+	"github.com/docker/docker/pkg/ioutils"
```

Comment:
is this reference intended?

Reply:
Hm. No, I don't think so. Thanks for pointing out!